src/reference/00-Getting-Started/02-Hello.md

```diff
@@ -4,72 +4,77 @@ out: Hello.html
 
   [Basic-Def]: Basic-Def.html
   [Setup]: Setup.html
+  [Running]: Running.html
 
 Hello, World
 ------------
 
 This page assumes you've [installed sbt][Setup].
 
-### Create a project directory with source code
+### sbt new command
```

Comment:
Would it make sense to simply say 

> This page assumes you've installed sbt 0.13.10 or later.

Since there are is no guidance given for creating a project by hand? This seems fine, although [`sbt new` does not work for me](https://github.com/foundweekends/giter8/issues/228) and I have a hard time believing I'm the only person with this problem. We'll find out I guess.